lib/apartmentex/tenant_actions.ex

```diff
@@ -34,7 +34,7 @@ defmodule Apartmentex.TenantActions do
   def create_schema(repo, tenant) do
     prefix = build_prefix(tenant)
     case repo.__adapter__ do
-      Ecto.Adapters.Postgres -> Ecto.Adapters.SQL.query(repo, "CREATE SCHEMA #{prefix}", [])
+      Ecto.Adapters.Postgres -> Ecto.Adapters.SQL.query(repo, "CREATE SCHEMA \"#{prefix}\"", [])
```

Comment:
This seems to be how ecto sends sql too, (with the quotes) we may also consider catching prefixes starting with pg to return a more informative error